target/linux/realtek/files-5.15/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -1848,13 +1853,34 @@ void rtl9300_sds_tx_config(int sds, phy_interface_t phy_if)
 
 	switch(phy_if) {
 	case PHY_INTERFACE_MODE_1000BASEX:
+		pre_amp = 0x1;
+		main_amp = 0x9;
+		post_amp = 0x1;
+		pre_en = 1;
+		post_en = 1;
```

Comment:
Setting the defaults for `pre_en` and `post_en` again?

In any case this whole switch case has a weird mix of hex and decimal notation...

Reply:
Both fixed in updated patches.